Docs 6155 choice options v4.2 (#1095)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/choice-router-concept.adoc

> . In the window that opens, log in to Anypoint Exchange and search for the example project Content-Based Routing.

Reviewer: Edit Instructions:
Hi Kim, could you please review the new content in this PR:

</mule>
----

> == Adding Additional Route Options

Reviewer: ```suggestion
== Adding Route Options
```

Reviewer: Actually, now that I see the subtopic, I suggest two Level-2 topics:
==Add Routes by Dragging Components in Studio
==Add Routes by Editing the XML
Maybe they should be Level-3 topics within a Level-2 topic named "Adding Routes to the Choice Router," which would have the text explaining the typical case of two routes and then why you might want to add others?

> By default, the Choice router has one route option and a default route. To add additional routes to the Choice router, follow these steps:

Reviewer: Clarify "has one route option and a default route."
I am guessing that this means that the Choice router has a default route (determined by what or who?) and provides you the ability to specify one other route (rather than changing the default?). And you perform this task if you want to have more routes available than the usual two (default plus...one other specified by...?).

Reviewer: ```suggestion
By default, the Choice router has one route option and a default route. To add routes beyond these two to the Choice router, follow these steps:
```

. From *Mule Palette* in Anypoint Studio, click and hold any component.

> . Drag the component inside the Choice router where you see a vertical black line:

Reviewer: I think we want to avoid "click and hold":
. From *Mule Palette* in Anypoint Studio, click any component and drag it into the Choice router, where you see a vertical black line:

+
image:mruntime-choice-add-route.png[add+route+option]

> === Add Additional Routes by Editing the XML

Reviewer: See previous title suggestion.

> You can also add additional route options to a Choice router by modifying the XML config file

Reviewer 2: Add missing : at the end.
_You can also add additional route options to a Choice router by modifying the XML config file:_

> You can also add additional route options to a Choice router by modifying the XML config file:

Reviewer: ```suggestion
You can also add additional route options to a Choice router by modifying the XML configuration file:
```

> . In Studio, right click the Choice router and select *Go to XML...*.

Reviewer: ```suggestion
. In Studio, right-click the Choice router and select *Go to XML...*.
```

> . Inside the `<choice>` element, add a `<when>` element.

Reviewer: ```suggestion
. Inside the `<choice>` element, add a `<when>` element:
```
